### ISSUE TITLE
update code to pytorch 1.9 with new torch.fft module

### ISSUE THREAD

**Issue by giovp** [SteffenCzolbe/PerceptualSimilarity#18]
Hi,

thanks a lot for this repo, really well documented! I was about to try out the watson-dft loss with pytorch 1.9 but encountered an error at this line:

https://github.com/SteffenCzolbe/PerceptualSimilarity/blob/3729dd1fc7e7082c0c66f7dc74d119a78948cb17/src/loss/rfft2d.py#L53

A bit of search led me to the new torch.fft module in 1.7 (which was then removed from 1.8 onwards, here's a nice wiki from the torch team: https://github.com/pytorch/pytorch/wiki/The-torch.fft-module-in-PyTorch-1.7).

Playing around a bit, I managed to simply replace the above line with the following (with defaults args).

```
coeff = torch.view_as_real(torch.fft.rfft2(x))
```

I was just wondering if you could double check that this is the correct way to update the rfft2 blockwise operations with the new torch.fft module.

Thank you!

**Comment by SteffenCzolbe** [comment]
Updating the torch version is currently not in scope. DL code ages quickly, and I simply do not have the time to update all my projects.

The code should work as-is with the library versions listed in the requirements.txt file.

**Comment by giovp** [comment]
right ok, understand. Just wanted to mention that that one is the only offending line I found for the `watson-dft` and was wondering if my proposed solution is consistent in your opinion.

**Comment by mdda** [comment]
FWIW, I've forked the repo and added your `rfft2`, plus also adjusted some of the imports so that it works for more modern PyTorch / python.  

I've only tested it for the loss function import sample (also updated), though, so YMMV.
